SwrveSDK.podspec

```diff
@@ -17,6 +17,7 @@ Pod::Spec.new do |s|
   s.dependency 'SwrveSDKCommon', '4.5.2'
   s.dependency 'SwrveConversationSDK', '4.5.2'
 
-  s.frameworks = 'UIKit', 'QuartzCore', 'CFNetwork', 'StoreKit', 'Security', 'CoreTelephony', 'MessageUI', 'CoreLocation', 'AddressBook', 'AVFoundation', 'AssetsLibrary', 'Contacts', 'Photos'
+  s.frameworks = 'UIKit', 'QuartzCore', 'CFNetwork', 'StoreKit', 'Security', 'CoreTelephony', 'MessageUI', 'CoreLocation', 'AddressBook', 'AVFoundation', 'AssetsLibrary', 'Photos'
+  s.weak_frameworks = 'Contacts'
```

Comment:
I'm looking into it here. still some compatibility issues that I'm going to work through for iOS7